snaps/how-to/work-with-existing-snaps.md

```diff
@@ -0,0 +1,31 @@
+# Working with third-party snaps
+
+While some snaps are designed to be one-off and dedicated one and only one Dapp, this isn't always the case. Third-party developers like yourself can use these snaps to build your own Dapps, or even to compose with another snap.
+
+This is possible because [snaps can expose an RPC API](../reference/exports/#onrpcrequest). Snaps can decide to make their API available to other Dapps, other snaps, or both, by requesting the [`endowment:rpc`](../reference/permissions.md#endowmentrpc) permission.
+
+## Installing a snap
+
+Installing a snap is done by using the [`wallet_requestSnaps`](../reference/rpc-api.md#wallet_requestsnaps) method from your Dapp or your snap.
+
+:::tip
+A snap is not installed only once. It has to be installed for every consumer that wants to use it. For example, if you're building two Dapps that use the same snap, each Dapp will need to install the snap independently.
```

Comment:
The snap is only installed one time when the first dApp requests the snap. All the other dApp that wants to use the snap will just need to request permission to talk to the snap

Reply:
I think we decided to use `website` instead of `dapp`, but could be wrong.

cc @coreyjanssen and maybe @rekmarks

Reply:
You're correct, @eriknson! We've opted to say `site` instead of `dapp`, as it's more user friendly.